stable/ibex-dashboard/README.md

```diff
@@ -0,0 +1,81 @@
+# nginx-ingress
+
+[Ibex Dashboard](https://github.com/CatalystCode/ibex-dashboard) is a dashboarding application that enables building dashboard and templates. It mainly supports Application Insights but data sources and visual components are easily extendable.
+
+## TL;DR;
+
+```console
+$ helm install stable/ibex-dashboard
+```
+
+## Introduction
+
+This chart bootstraps an ibex-dashboard deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+  - Kubernetes 1.4+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release stable/ibex-dashboard
+```
+
+The command deploys ibex-dashboard on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the Ibex chart and their default values.
+
```

Comment:
Please format the table properly. Either align the columns correctly or remove multiple spaces completely.